.husky/pre-commit

```diff
@@ -7,5 +7,8 @@ npx --no-install lint-staged
 # lint only staged ruby files
 git diff --name-only --cached | xargs ls -1 2>/dev/null | grep '\.rb$' | xargs bundle exec rubocop --force-exclusion -a
 
+# echo git-sha to file
```

Comment:
@vishnu-narayanan Can we make this get this dynamically in the initializer? By making it a commit hook, we would need to ensure that the git-sha is updated every time a PR is raised.





Can we do something like this? https://stackoverflow.com/a/33380485

```
running_rake = ::File.basename($0) == 'rake'

unless running_rake
  latest_commit = `git rev-parse HEAD`
  STDOUT.puts "**** latest commit: #{latest_commit}"
end
```

Reply:
@pranavrajs Gotcha. Will make the changes.